mempool/mempool_test.go

```diff
@@ -0,0 +1,92 @@
+package mempool_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/NethermindEth/juno/blockchain"
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/db"
+	"github.com/NethermindEth/juno/db/pebble"
+	"github.com/NethermindEth/juno/mempool"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMempool(t *testing.T) {
+	testDb := pebble.NewMemTest(t)
+	pool := mempool.New(testDb)
+	blockchain.RegisterCoreTypesToEncoder()
+
+	t.Run("empty pool", func(t *testing.T) {
```

Comment:
It might be better to forego the sub-tests in this case, since we can't run them individually, i.e., 
```
go test -run 'TestMempool/push_multiple_to_empty'
```
The alternative is to refactor the sub-tests to make them stateless, but I'll leave it to you to decide if that's worth it.